lib/systems/examples.nix

```diff
@@ -23,6 +23,7 @@ rec {
 
   ppc64-elfv1 = {
     config = "powerpc64-unknown-linux-gnuabielfv1";
+    rust.rustcTarget = "powerpc64-unknown-linux-gnu";
```

Comment:
Should the same be added for elfv2?

Reply:
The rust prebuilt archive for bootstrapping is ELFv1, and the different ELF versions use differently-named dynamic linker names, so I think the archive wouldn't be runnable in an ELFv2 stdenv and some cross setup would need to be done?

Also, it looks like our `rustc` can't generate ELFv2 object code on glibc powerpc64 currently anyway: https://github.com/NixOS/nixpkgs/issues/416907